graphql_compiler/query_pagination/query_parameterizer.py

```diff
@@ -60,10 +64,18 @@ def _get_filter_node_operation(filter_directive: DirectiveNode) -> str:
     return cast(StringValueNode, filter_directive.arguments[0].value).value
 
 
-def _is_new_filter_stronger(operation: str, new_filter_value: Any, old_filter_value: Any) -> bool:
+def _is_new_filter_stronger(
+    query_analysis: QueryPlanningAnalysis,
+    property_path: PropertyPath,
+    operation: str,
+    new_filter_value: Any,
+    old_filter_value: Any,
```

Comment:
nit: These are expected to be the same type, right? That's what I gather from the docstring at least. If so, `new_filter_value: T, old_filter_value: T` may be more appropriate.

Reply:
Not sure if that's changing anything. Can't mypy just plug in `object` for `T`? Strong typing with inheritance is weird.